network_files/config/client.toml

```diff
@@ -0,0 +1,17 @@
+# This is a TOML config file.
+# For more information, see https://github.com/toml-lang/toml
+
+###############################################################################
+###                           Client Configuration                            ###
+###############################################################################
+
+# The network chain ID
+chain-id = ""
+# The keyring's backend, where the keys are stored (os|file|kwallet|pass|test|memory)
+keyring-backend = "test"
```

Comment:
KEYRING should not be test. We should be using and encouraging all operators to use OS, kwallet, or maybe file backends